handlers/report/report.go

```diff
@@ -62,13 +75,13 @@ func (h *reportHandler) getWeatherReport(w http.ResponseWriter, r *http.Request)
 
 // parseLatLon parses the lat and lon from request query params
 func parseLatLon(params url.Values) (lat float64, lon float64, err error) {
-	if lp, ok := params[LAT_PARAM]; ok {
+	if lp, ok := params[LAT_PARAM]; ok && len(lp) > 0 {
```

Comment:
Not sure if passing `?lat=` would be read as `"lat": ""` or `"lat": nil`, but might as well guard against an index out of bounds error just in case.